### PR TITLE
Fix for AWS Lambda integration feature not working

### PR DESCRIPTION
**Pull request by sanjulamadurapperuma**
Fixes [wso2/product-apim#7529](https://github.com/wso2/product-apim/issues/7529)

image

image

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/AWSLambda/Credentials.jsx [hunk 1]
```diff
@@ -190,6 +279,37 @@ export default function Credentials(props) {
                         endpointsDispatcher({ action: 'set_awsCredentials', value: newEndpointConfig });
                     }}
                 />
+                <FormControl
+                    required
+                    margin='normal'
+                    variant='outlined'
+                    disabled={endpointConfig.access_method === 'role-supplied'}
+                >
+                    <InputLabel ref={inputLabel} id='demo-simple-select-outlined-label'>
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/AWSLambda/Credentials.jsx]: Shall we remove this id if not in use, or change it to a proper name `demo-simple-select-outlined-label`

Comment (sanjulamadurapperuma) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/AWSLambda/Credentials.jsx]: Sure, I think that id can be removed. Thanks for spotting it!

File: features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/fr.json [hunk 2]
```diff
@@ -654,7 +657,23 @@
   "Apis.Details.Resources.components.AddOperation.clear.inputs.tooltip": "",
   "Apis.Details.Resources.components.AddParameter.add.tooltip": "",
   "Apis.Details.Resources.components.AddParameter.clear.inputs.tooltip": "",
+  "Apis.Details.Resources.components.operationComponents.\n                                OperationGovernance.operation.scope.create.new.scope": "",
```
Comment (menakaj) [inline features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/fr.json]: Shall we remove these new line in the key? Using string concatenation will help to do that.

Comment (sanjulamadurapperuma) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/fr.json]: Sure

Comment (sanjulamadurapperuma) [reply features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/public/locales/fr.json]: They have been removed as requested

**Pull request merged**